doc/040_backup.rst

```diff
@@ -50,7 +50,24 @@ still get a nice live status display. Be aware that the live status shows the
 processed files and not the transferred data. Transferred volume might be lower
 (due to de-duplication) or higher.
 
-If you run the command again, restic will create another snapshot of
+On Windows, the ``--use-fs-snapshot`` option will use Windows Volume Shadow Copy
```

Comment:
It's very critical to the functionality of this new feature that we add a `'` after "Windows" ;)

Reply:
I'm not really sure about the grammar because "Windows" is singular in this case.

Reply:
Your input on this made me look into it, and I found that there doesn't seem to be a global concensus of whether or not to use just `'` or `'s` to denote possession with a singular noun that ends in `s`. It's pretty clear that there should be an apostrophe though, what hasn't been agreed upon is whether it should be followed by a `s`.

[Here's](https://www.dictionary.com/e/whats-the-rule-for-doing-a-possessive-after-the-word-s/) a good article that sums it all up. It references both Oxford and APA which are both prominent sources for this stuff.

In conclusion, one can use any of the above, as long as one is consistent within the same article. One rule that could be followed to me more specific is the "one-syllable rule" mentioned in the article, which means:

- When the noun has just one syllable, add `'s`.
- When the noun has multiple syllables, add just `'`.

I think that makes sense, and would rather follow something that's more specific than "pick whichever way you want". Anyway there's supposed to be an apostrophe, with or without subsequent `s`. I suggest adding a single `'` after "Windows". Any takers? :)

Reply:
Because you put that much effort into researching this I think your interpretation is correct.

Fixed in https://github.com/restic/restic/pull/2274/commits/0bb800fca92c46875b74458cf84baf515b6a1da5

Reply:
Hehe, thanks. I just find it interesting :)

Reply:
I've also had a discussion about the grammar but lacked the time to do the research. ;-)